src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.EventWaitHandle.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        internal const uint CREATE_EVENT_INITIAL_SET = 0x2;
+        internal const uint CREATE_EVENT_MANUAL_RESET = 0x1;
+
+        [DllImport(Interop.Libraries.Kernel32, SetLastError = true)]
```

Comment:
Could you please delete the interop definitions that are no longer used from src\Common\src\Interop\Windows\mincore\Interop.Threading.cs